container/falcon-container-aks-implementation-guide.md

```diff
@@ -0,0 +1,282 @@
+# Implementation Guide for CrowdStrike Falcon-Container Sensor on Azure AKS Kubernetes cluster
+
+This guide works through creation of a new Kubernetes cluster, deployment of the Falcon-Container Sensor, and demonstration of detection capabilities of Falcon Container Workload Protection.
+
+No prior Kubernetes or Falcon knowledge is needed to follow this guide. First sections of this guide focus on creation of ACR Container registry and an AKS cluster, however, these sections may be skipped if you have access to an existing cluster.
+
+Time needed to follow this guide: 45 minutes.
+
+## Pre-requisites
+
+- Refer to the lastest documentation on the [falcon-container](https://falcon.crowdstrike.com/documentation/146/falcon-container-sensor-for-linux)
+- Existing Azure Subscription and Global Adminstrator permissions
+- You will need a workstation to complete the installation steps below
+  * These steps have been tested on Linux and should also work with OSX
+- Install [docker](https://www.docker.com/products/docker-desktop) container runtime
+- Verify docker daemon is running on your workstation
```

Comment:
Would it improve clarity by saying something like "A workstation or virtual machine running the Docker container runtime" ?